tools/generators/version-bump/index.spec.ts

```diff
@@ -206,6 +213,34 @@ describe('version-string-replace generator', () => {
         }
       `);
     });
+
+    it('should exclude packages', async () => {
+      await generator(tree, {
+        all: true,
+        ...defaultTestOptions,
+        exclude: '@proj/react-utilities,@proj/react-theme',
+      });
+
+      const reactThemePackageJson = readJson(tree, 'packages/react-theme/package.json');
+      const makeStylesPackageJson = readJson(tree, 'packages/make-styles/package.json');
+      const reactButtonPackageJson = readJson(tree, 'packages/react-button/package.json');
+      const reactUtilitiesPackageJson = readJson(tree, 'packages/react-utilities/package.json');
+      expect(reactButtonPackageJson.version).toMatchInlineSnapshot(`"9.0.0-beta.0"`);
```

Comment:
I wonder if it would be better to test against explicit strings rather than inline snapshots for some of these?

Reply:
yeah makes sense, I'll do that. I'll keep the object snapshots for simplicity